go-controller/pkg/ovn/gress_policy.go

```diff
@@ -65,6 +71,10 @@ func (pp *portPolicy) getL4Match() (string, error) {
 }
 
 func newGressPolicy(policyType knet.PolicyType, idx int, namespace, name string) *gressPolicy {
+	ipBlock := ipBlock{
+		except: make([]string, 0),
```

Comment:
@JacobTanenbaum I think you can just leave this empty; a nil array should work fine here. In fact, you can just do `&ipBlock{},` below.

Reply:
done